sycl/doc/extensions/SPIRV/SPV_INTEL_arbitrary_precision_floating_point.asciidoc

```diff
@@ -1115,21 +1117,23 @@ _RoundingAccuracy_ controls the accuracy of the rounding operation and is chosen
 
 | Capability:
 *ArbitraryPrecisionFloatingPointINTEL*
-| 8 | 5842 | <id> Result Type | Result <id> | A <id> | _Literal_ Mout | _Literal_ EnableSubnormals | _Literal_ RoundingMode | _Literal_ RoundingAccuracy
+| 8 | 5842 | <id> Result Type | Result <id> | A <id> | _Literal_ Mout | _Literal_ FromSign | _Literal_ EnableSubnormals | _Literal_ RoundingMode | _Literal_ RoundingAccuracy
 |=====
 
 [cols="9", width="100%"]
```

Comment:
The same here: Cols number for this table and number of SPIR-V words should be increased by 1.